BrowserSelect/Program.cs

```diff
@@ -62,7 +63,7 @@ static void Main(string[] args)
                     var browser = sr[1];
 
                     // matching the domain to pattern
-                    if (DoesDomainMatchPattern(uri.Host, pattern))
```

Comment:
If you don't use it here anymore, `DoesDomainMatchPattern` becomes dead code. Please remove any methods that aren't used anywhere in the codebase.